docs/plain-bundle-spec.md

```diff
@@ -0,0 +1,95 @@
+# Plain Bundle Spec
+
+## Overview
+
+This document is meant to define the plain bundle format as a reference for those publishing plain bundles for use with
+RukPak. A bundle is a collection of Kubernetes resources that are packaged into a container image. These bundles can be
+unpacked onto a running cluster, where controllers can then create the underlying content embedded in the bundles.
+
+Plain bundles are simply container images containing a set of static, arbitrary, Kubernetes YAML manifests in a given
+directory. The currently implemented plain bundle schema is the `plain+v0` format. The name of the bundle
+format, `plain+v0` combines the type of bundle (plain) with the current schema version (v0).
```

Comment:
Where is the current schema version is defined (or specified)?

Reply:
This is a good question. I'm not sure that we have a formal definition anywhere of how we version the schema. The proposed schema version `v0` is meant to indicate that the schema version is experimental and still not fully set in stone. We can clarify this more in the doc.